packages/auto-approve/src/get-pr-info.ts

```diff
@@ -53,6 +53,7 @@ export async function getChangedFiles(
       logger.error(
         `Not found error, ${err.status}, ${err.message} for ${owner}/${repo}/${prNumber}`
       );
+      return undefined;
```

Comment:
Is this a setup issue for the bot (not requesting enough permissions)? Shouldn't it have permissions to read the code in the PR?

Reply:
I think some repos in the org have more restrictions than others, the current set of permissions seem to be enough for the repos that use auto-approve.